packages/apps-config/src/endpoints/testing.ts

```diff
@@ -192,6 +192,13 @@ export function createTesting (t: TFunction): LinkOption[] {
         'KILT Protocol': 'wss://full-nodes.kilt.io:9944/'
       }
     },
+    {
+      info: 'kylin',
+      text: t('testnet.kylin-node.co.uk', 'Kylin Testnet', { ns: 'apps-config' }),
+      providers: {
+        'Kylin Network': 'wss://testnet.kylin-node.co.uk'
+      }
+    },
```

Comment:
The same params are used here - https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/endpoints/testingRelayRococo.ts#L245-L252

Is this a parcahain on Rococo or a stand-alone network?